homework-submission/react-hyf/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "react-hyf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "concurrently": "^4.1.1",
```

Comment:
you have a `concurrently` package with no use, it will take more memory and disk space.

Reply:
I set up concurrently properties and it is working properly now.